domdiv/main.py

```diff
@@ -990,12 +1010,22 @@ def filter_sort_cards(cards, options):
                     if 'short_name' in set_values:
                         exp_name = set_values['short_name']
 
+                card_names = []
+                for key, n in sorted(cardnamesByExpansion[exp].items(), key=lambda (k, (x)): x['sort']):
```

Comment:
You don't need any parens around the `lambda` args.

Reply:
I'll fix that.  It was a hold over from a point in time when I was having issues with the lambda function.